recipes/monsda/meta.yaml

```diff
@@ -17,32 +17,32 @@ build:
     - monsda = MONSDA.RunMONSDA:main
     - monsda_configure = MONSDA.Configurator:main
   noarch: python
+  run_exports:
+    - {{ pin_subpackage("monsda", max_pin="x") }}
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.11.4
     - pip
     - setuptools
     - pytest-runner
   run:
-    - biopython>=1.79
-    - grep>=3.4
-    - libmamba>=0.21.2
-    - mamba>=0.17.0
-    - more-itertools>=8.8.0
-    - natsort>=7.1.1
-    - nextflow
-    - numpy>=1.21.0
-    - pandas>=1.4.1
-    - perl>=5.32.1
-    - python>=3.9
-    - scipy>=1.7.0
-    - snakemake-minimal>=6.5.3
-    - snakemake>=6.5.3
-    - versioneer>=0.20
-    - yaml>=0.2.5
-    - zlib>=1.2.11
-    - zstd>=1.5.0
+    - grep >=3.4
+    - conda-libmamba-solver >=23.5.0
```

Comment:
This should not be needed and also not part of the environment.

Reply:
True, thanks